src/main/java/cz/muni/fi/timeline/service/StudyGroupService.java

```diff
@@ -2,12 +2,52 @@
 
 import cz.muni.fi.timeline.entity.StudyGroup;
 import cz.muni.fi.timeline.entity.User;
+import org.springframework.stereotype.Service;
 
+import java.util.Optional;
+import java.util.Set;
+
+/**
+ * @author Matej Macák
+ */
+@Service
 public interface StudyGroupService {
+    /**
+     * @param studyGroup study group to be created
+     * @param teacher teacher who created the study group
+     */
     void createStudyGroup(StudyGroup studyGroup, User teacher);
```

Comment:
Nakonec bych bral jako vstup asi jen studygroup. A do StudyGroup classy přidáme "NotNull" field owner.

Reply:
Těžko říct co bude nejlepší; to musíme ještě pořešit.